src/tools/attack/attack.cpp

```diff
@@ -458,7 +458,10 @@ main(
         };
         // flag
         QUIC_EXECUTION_CONFIG_FLAGS Flags = QUIC_EXECUTION_CONFIG_FLAG_XDP;
-        Flags |= AttackType == 0 ? QUIC_EXECUTION_CONFIG_FLAG_QTIP : QUIC_EXECUTION_CONFIG_FLAG_NONE;
+
+        // TODO: Since we moved the QTIP configuration from the datapath to the core layer (MsQuic Settings),
+        //       and this tool seems to be using purely the datapath layer, do we just remove QTIP support from here?
+
```

Comment:
Let's just remove support.

Reply:
@ProjectsByJackHe I see that the PR was merged before these last comments have been addressed. Are you planning a follow up PR?

Reply:
Yes, I have a follow up PR enqueued.

Reply:
https://github.com/microsoft/msquic/pull/4961